newswires/client/src/WireItemList.tsx

```diff
@@ -75,17 +84,26 @@ export const WireItemList = ({
 function decideMainHeadingContent(
 	supplier: string,
 	{ headline, slug }: WireData['content'],
+	hasDataFormatting: boolean,
 ): string {
-	const prefix =
-		(supplier === 'AAP' || supplier === 'PA') && slug && slug.length > 0
-			? `${slug} - `
-			: '';
+	const hasNonEmptySlug = slug && slug.length > 0;
 
 	if (headline && headline.length > 0) {
+		/**
+		 * AAP and PA stories have useful slugs. But stories with 'data formatting' have
+		 * their slugs added to their headlines when we get them, so we don't need to add them again.
+		 */
+		const prefix =
+			(supplier === 'AAP' || supplier === 'PA') &&
```

Comment:
I feel like these conditions could be abstracted out to perhaps a `showPrefix` boolean to make this ternary a bit simpler to parse

Reply:
good call, thanks! I've done that now 👍